miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AssayTestRestControllerST.java

```diff
@@ -0,0 +1,125 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.AssayTest;
+import uk.ac.bbsrc.tgac.miso.dto.AssayTestDto;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.AssayTest.PermittedSamples;
+import org.springframework.security.test.context.support.WithMockUser;
+import static org.junit.Assert.*;
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class AssayTestRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/assaytests";
+  private static final Class<AssayTest> entityClass = AssayTest.class;
+
+  private List<AssayTestDto> makeCreateDtos() {
+    AssayTest atest1 = new AssayTest();
+    atest1.setLibraryQualificationMethod(AssayTest.LibraryQualificationMethod.LOW_DEPTH_SEQUENCING);
+    atest1.setPermittedSamples(AssayTest.PermittedSamples.ALL);
+
+    AssayTestDto test1 = AssayTestDto.from(atest1);
```

Comment:
Create the DTOs directly. You can set the fields in the DTOs to the enum value's `.name()`